index.html

```diff
@@ -1,29 +1,22 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8">
-    <title>Fork Me! FCC: Test Suite Template</title>
-    <link rel="stylesheet" href="css/style.css">
-  </head>
-
-  <body>
-    <h1>Hello World</h1>
-
-
-    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
-
-    <!--
-
-    Hello Camper!
-
-    For now, the test suite only works in Chrome!
-    Please read the README below in the JS Editor before beginning.
-    Feel free to delete this message once you have read it.
-    Good luck and Happy Coding!
-
-    - The freeCodeCamp Team
-
-    -->
-    <script  src="js/index.js"></script>
-  </body>
-</html>
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./css/style.css">
+  <title>Microverse</title>
+</head>
+<body>
+  <header>
+    <h1>First Microverse Project</h1>
+  </header>
+  <main>
+    <section>
+      <article>
+        <p>I'm a software Developer</p>
```

Comment:
Please make sure to capitalize keywords in your sentence "software" is supposed to start with a capital letter.

Reply:
Great job @tonnymuchui 🤝 

- [ ] [Optional] Kindly use emojis for your comments because they make your communication human-friendly.
- [ ] [Optional] Always explain why you give a certain comment.